plex/provider_observer.py

```diff
@@ -93,7 +93,6 @@ def Start(self, mediaProvider: xbmcmediaimport.MediaProvider):
             raise ValueError('invalid mediaProvider')
 
         self._actions.append((ProviderObserver.Action.Start, mediaProvider))
-        log(f"started observation of {mediaProvider.friendlyName}", xbmc.LOGINFO)
```

Comment:
Any reason why this log is removed?

Reply:
That is one I added while going through, mediaProvider doesn't have friendlyName at this point yet so it throws an exception, so removed it.